content/en/docs/concepts/configuration/pod-overhead.md

```diff
@@ -26,31 +26,138 @@ on top of the container requests & limits.
+field within the PodSpec.
+
+After the RuntimeClass admission controller, the PodSpec will show:
+
+```bash
+$ kubectl get pod test-pod -o json | jq '.spec.overhead'
+{
+  "cpu": "250m",
+  "memory": "120Mi"
+}
+```
+
+If a ResourceQuota is defined, the sum of container requests as well as the
+Overhead field are counted.
+
+When kube-scheduler runs, the Overhead as well as sum of container requests are taken into account when identifying
+an appropriate node binding, again seeking a node which has 2.25 CPU and 320 MB of memory available.
+
+Once scheduled for the node, as part of the Pod creation process Kubelet will create a Pod cgroup (for non burstable QoS)
+and this will be sized as the sum of container (requests or limits) plus the Overhead defined in the PodSpec. 
+
+Taking a look at the container requests, we see a total of 2000m CPU and 200Mi:
+```bash
+$ kubectl get pod test-pod -o json | jq '.spec.containers[].resources.limits'
+{
+  "cpu": "500m",
+  "memory": "100Mi"
+}
+{
+  "cpu": "1500m",
+  "memory": "100Mi"
+}
+```
+
+From a node perspective, it sees that 2250m CPU and 320Mi are being requested:
+
+```bash
+$ kubectl describe node | grep test-pod -B2
+  Namespace                   Name                CPU Requests  CPU Limits   Memory Requests  Memory Limits  AGE
+  ---------                   ----                ------------  ----------   ---------------  -------------  ---
+  default                     test-pod            2250m (56%)   2250m (56%)  320Mi (1%)       320Mi (1%)     36m
+```
+
+This is reflected in the cgroups on the node where the owrkload is running. Kubelet creates the Pod cgroup
+with the Pod Overhead taken into account, and it is within this cgroup that the containers run, whether it be through
+containers running inside of a virtual machine or via namespaces containers running on the host.
+
+The relevant memory cgroups for the example workload are shown below:
+```bash
+root@kind-worker2:~# cat /sys/fs/cgroup/memory/kubepods/pod083eaf28-d8ff-4e1b-b9eb-46d45fffee95/memory.limit_in_bytes
+335544320
+
+root@kind-worker2:~# cat /sys/fs/cgroup/memory/kubepods/pod083eaf28-d8ff-4e1b-b9eb-46d45fffee95/**/memory.limit_in_bytes
+104857600
+9223372036854771712
+104857600
+```
+
+Ignoring the pause container, which doesn't have a limit set (9223372036854771712), the containers are each 100 MB
+and the Pod cgroup is set to 320 MB, as expected.
+
+### Observability
+
+A `kube_pod_overhead` metric is available in [kube-state-metrics](https://github.com/kubernetes/kube-state-metrics) to help identify when `PodOverhead` is
+being utilized and to help observe stability of workloads running with a defined Overhead.
+This functionality is not available in the 1.9 release of kube-state-metrics, but is expected
+in the next release. Users will need to build kube-state-metrics from source in the meantime.
```

Comment:
[Avoid statements about the future](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-statements-about-the-future)

Reply:
In this case, I'm highlighting that post 1.9 this should be expected.  I could just say not available in 1.9, but I think setting expectation that 2.0 or 1.10 should include *may* be helpful?